azure/lerna.json

```diff
@@ -1,4 +1,5 @@
 {
-	"packages": ["packages/**"],
-	"version": "1.2.0"
+	"version": "1.2.0",
+	"npmClient": "pnpm",
+	"useWorkspaces": true
```

Comment:
Now that workspaces are enabled, what are the implications for devs here? If we already have this documented somewhere, just a pointer would be sufficient.

Reply:
Since we treated release groups as a unit already, there's very few implications for devs. We have docs on dep management [in the wiki](https://github.com/microsoft/FluidFramework/wiki/Managing-dependencies). I'll add a link to the readme.